### PR TITLE
Update uma-title.csv

### PR DESCRIPTION
**Pull request by latoulicious**
Adding few translated Horse Title.

### CODE REVIEW

**Comment by FabulousCupcake** [comment]
@robflop Could you review this 👀

File: src/data/uma-title.csv [hunk 1]
```diff
@@ -59,25 +59,25 @@
 "伝説の歌姫","Legendary Diva"
 "スキルマスター","Skill Mastery"
 "技のデパート","Various Technique"
-"日本の総大将",""
+"日本の総大将","General of Japanese"
```
Comment (robflop) [inline src/data/uma-title.csv:62]: Supreme Commander of Japan

File: src/data/uma-title.csv [hunk 2]
```diff
@@ -59,25 +59,25 @@
...
 "異次元の逃亡者","Dimensional Runaway"
 "怪鳥","Eerie Bird"
 "スーパーカー","Supercar"
 "変幻自在","Shape Shifter"
 "驀進王","Bakushin・O"
 "高速ステイヤー","High Speed Stayer"
 "不死鳥","Phoenix"
-"常識破りの女帝",""
+"常識破りの女帝","The Empress of Common Sense"
```
Comment (robflop) [inline src/data/uma-title.csv:70]: Empress of Breaking Common Sense

File: src/data/uma-title.csv [hunk 3]
```diff
@@ -59,25 +59,25 @@
...
 "ミスパーフェクト","Miss Perfect"
 "帝王","Teio・Emperor"
 "破天荒","Unheard"
 "皇帝","Kotei・Emperor"
 "笑門来福",""
-"愛しき名脇役",""
+"愛しき名脇役","Lovely Supporting Role"
```
Comment (robflop) [inline src/data/uma-title.csv:76]: Beloved Supporting Actor

(Although the japanese includes both "Beloved" and "Excellent", "Beloved Excellent Supporting Actor" is very weird in english, so restricting it to Beloved seems fine to me)

File: src/data/uma-title.csv [hunk 4]
```diff
@@ -59,25 +59,25 @@
...
 "麗しき実力者",""
-"最強マイラー",""
+"最強マイラー","Strongest Miler"
 "アイドルウマ娘","Idol Girl"
-"世紀末覇王",""
+"世紀末覇王","End of Century Overlord"
```
Comment (robflop) [inline src/data/uma-title.csv:80]: Supreme Ruler of the Century's End

(Or something more concise)

File: src/data/uma-title.csv [hunk 5]
```diff
@@ -95,7 +95,7 @@
 "変則二冠ウマ娘",""
 "パワフルウマ娘","Powerful Girl"
 "完全無欠","Flawless"
-"粗削りのマイル王",""
+"粗削りのマイル王","King of the Rough Miles"
```
Comment (robflop) [inline src/data/uma-title.csv:98]: Unrefined Mile King

**Review by robflop — CHANGES_REQUESTED** [review]

**Comment by FabulousCupcake** [comment]
Thank 🙇 

@latoulicious what do you think? Please apply/reject/comment the suggestions, I'll merge the PR after all is done

**Comment by latoulicious** [comment]
Well that sound good enough for me. yea i was thinking some of my translation look weird but it based on the actual history of the horse. thanks for the correction.

File: src/data/uma-title.csv (hunk 4 above)
Comment (latoulicious) [reply src/data/uma-title.csv:80]: well, based on the japanese people used to call TM Opera O with "End of Century Overlord/King of the end century"

File: src/data/uma-title.csv (hunk 1 above)
Comment (latoulicious) [reply src/data/uma-title.csv:62]: I'm not pretty sure with the translation before , its weird to translate that to english but thanks.

File: src/data/uma-title.csv (hunk 5 above)
Comment (latoulicious) [reply src/data/uma-title.csv:98]: King of the mile is also fine imo

File: src/data/uma-title.csv (hunk 4 above)
Comment (latoulicious) [reply src/data/uma-title.csv:80]: so i think that might be suitable for her

File: src/data/uma-title.csv (hunk 5 above)
Comment (robflop) [reply src/data/uma-title.csv:98]: King of the Mile leaves out the first word, 粗削り. So I wouldn't use that.

File: src/data/uma-title.csv (hunk 4 above)
Comment (robflop) [reply src/data/uma-title.csv:80]: I'm not knowledgeable on horse racing history, so i'd trust you there then.

File: src/data/uma-title.csv (hunk 5 above)
Comment (latoulicious) [reply src/data/uma-title.csv:98]: so , what you recommend for the title? well yes the "Rough" word didn't included

Comment (robflop) [reply src/data/uma-title.csv:98]: I recommend my initial suggestion, i'd expect another の if it was King of the Rough Miles.

**Review by FabulousCupcake — APPROVED** [review]

**Comment by FabulousCupcake** [comment]
Thank you!

**Pull request merged**